projects/security.md

```diff
@@ -0,0 +1,58 @@
+# Security Semantic Conventions Working Group
+
+## Description
+
+The purpose of this working group is to bring in the security domain for the OpenTelemetry community. 
```

Comment:
We should make sure to clearly distinguish this WG from the Security SIG in OTel (https://github.com/open-telemetry/community/issues/1454).

Reply:
This is especially important since we no longer use the term Working Group. Both would be SIGs: one ephemeral, one permanent. We should make sure that's reflected in naming wherever it's used.

Reply:
Let's have a follow up PR to replace working group with SIG

Reply:
https://github.com/open-telemetry/community/pull/2093